styles/rating/rating.css

```diff
@@ -0,0 +1,20 @@
+
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
+  --default: rgb(53, 52, 52);
+  --unchecked-rating:rgba(10, 4, 4, 0.267);
+}
```

Comment:
Keep all the variable declarations in a single file. because if you are declaring variables in every CSS file then it will overwrite the previous variables if the names are the same.

Reply:
sure, I will do that, it will be done for every component , so I will create a branch for it.

Reply:
sure, I will do that, this is done in every css file, so I will create a branch and make changes in all of them.